src/test/test_mempool.cc

```diff
@@ -406,7 +406,7 @@ TEST(mempool, btree_map_test)
 
 TEST(mempool, check_shard_select)
 {
-  const size_t samples = mempool::num_shards * 100;
+  const size_t samples = mempool::num_shards * 30;
```

Comment:
shouldn't it be` get_num_shards()` instead? I don't think the num_shards is public?

Reply:
Ack. Embarrassed that the commits didn't even compile :-( I clearly forgot to run ninja tests enough and didn't realize this wasn't being recompiled.